packages/cli/src/ui/utils/clipboardUtils.ts

```diff
@@ -8,109 +8,604 @@ import { exec } from 'node:child_process';
 import { promisify } from 'node:util';
 import * as fs from 'node:fs/promises';
 import * as path from 'node:path';
+import * as crypto from 'node:crypto';
+
+/**
+ * Interface for paste protection options
+ */
+export interface PasteProtectionOptions {
+  /** Check for sensitive data like API keys, tokens, etc. */
+  checkSensitiveData: boolean;
```

Comment:
please remove the sensitive data checks. that should be a separate PR and will need to be validated carefully to ensure false positives don't impact users.